Automattic/wp-calypso
# Stats: Moved Insights into component directory

This PR moves the `stats/insights.jsx` file to `stats/insights/index.jsx`. No SCSS comes along with it. pinging @timmyc 

**To Test**
- Open the Stats Insights page
- Ensure that everything looks the same :)
